submissions/2024-09-18-OpenFF-NAGL2-ESP-Timing-Benchmark-v1.1/compute2.json

```diff
@@ -0,0 +1,83 @@
+{
+  "qc_specifications": {
+    "wb97x-d/def2-TZVPPD/ddx-water": {
+      "method": "wb97x-d",
+      "basis": "def2-TZVPPD",
+      "program": "psi4",
+      "spec_name": "wb97x-d/def2-TZVPPD/ddx-water",
+      "spec_description": "wb97x-d functional with def2-TZVPPD basis set and DDX water (eps = 78.4) using UFF radii and 1.1x scaling factor",
+      "store_wavefunction": "none",
+      "implicit_solvent": {
+        "ddx_model": "pcm",
+        "ddx_radii_scaling": 1.1,
+        "ddx_radii_set": "uff",
+        "ddx_solvent_epsilon": 78.4,
+        "ddx_solvent": "water"
+      },
+      "maxiter": 200,
+      "scf_properties": [
+        "dipole",
+        "quadrupole",
+        "lowdin_charges",
+        "mulliken_charges",
+        "mbis_charges",
+        "mayer_indices",
+        "wiberg_lowdin_indices",
+        "dipole_polarizabilities"
+      ],
+      "keywords": {
+        "dft_spherical_points": 590,
+        "dft_radial_points": 99,
+        "dft_bs_radius_alpha": 5.0
+      }
+    },
+    "wb97x-d/def2-TZVPPD": {
+      "method": "wb97x-d",
+      "basis": "def2-TZVPPD",
+      "program": "psi4",
+      "spec_name": "wb97x-d/def2-TZVPPD",
+      "spec_description": "wb97x-d functional with def2-TZVPPD basis set and vacuum using UFF radii and 1.1x scaling factor",
+      "store_wavefunction": "none",
+      "implicit_solvent": null,
+      "maxiter": 200,
+      "scf_properties": [
+        "dipole",
+        "quadrupole",
+        "lowdin_charges",
+        "mulliken_charges",
+        "mbis_charges",
+        "mayer_indices",
+        "wiberg_lowdin_indices",
+        "dipole_polarizabilities"
+      ],
+      "keywords": {
+        "dft_spherical_points": 590,
+        "dft_radial_points": 99,
+        "dft_bs_radius_alpha": 5.0
+      }
+    }
+  },
+  "driver": "properties",
+  "priority": "normal",
+  "dataset_tags": [
+    "openff"
+  ],
```

Comment:
Checking the docs I'm not sure this winds up actually mattering, FYI -- as documented in #398 the only data that can be updated with compute.json is the compute spec, priority, and compute tag.

Reply:
Thanks, I'll add this info to the compute expansion instructions!